thirdparty/assimp/include/assimp/irrXMLWrapper.h

```diff
@@ -1,144 +0,0 @@
-/*
```

Comment:
Makes sense to remove indeed.

@fire Not directly related to this PR and could be done later on, but I see lots of included paths added in `modules/assimp/SCsub` that we don't have in our distribution (like most upstream `contrib` stuff which we don't use nor ship).